lib/android.js

```diff
@@ -720,7 +714,7 @@
~			} else if (message.length > 0) {
~				message += '\n';
~			}
~			message +=
~				__('Current installed Android SDK tools:') + '\n'
```

Comment:
<!--
  0 failure: 
  1 warning:  lib/android.js li...
  
  
  DangerID: danger-id-Danger;
  File: lib/android.js;
  Line: 782;
-->

- :warning: lib/android.js line 782 – Found non-literal argument to RegExp Constructor (security/detect-non-literal-regexp)